### PR TITLE
Add web-nfc tests about signal

### PR DESCRIPTION
**Pull request by xiuqijix**

### CODE REVIEW

File: web-nfc/NFCWriter_push.https.html [hunk 1]
```diff
@@ -94,13 +104,19 @@
...
 
-promise_test(t => {
+promise_test(async t => {
   const writer = new NFCWriter();
   const controller = new AbortController();
   const p = writer.push(test_text_data, { signal: controller.signal });
-  controller.abort();
-  return promise_rejects(t, 'NoModificationAllowedError', p);
-}, "NFCWriter.push should fail if the instance has already initiated NFC data transfer.");
+  const rejected = promise_rejects(t, 'NoModificationAllowedError', p);
```
Comment (Honry) [inline web-nfc/NFCWriter_push.https.html]: Should be "AbortError"

File: web-nfc/NFCWriter_push.https.html [hunk 2]
```diff
@@ -94,13 +104,19 @@
...
+  let callback_called = false;
+  t.step_timeout(() => {
+    callback_called = true;
+    controller.abort();
+  }, 10);
+  await rejected;
+  assert_true(callback_called, 'timeout should have caused the abort');
+}, "NFCWriter.push should fail if set signal's aborted flag when NFC data transfer.");
```
Comment (Honry) [inline web-nfc/NFCWriter_push.https.html]: This description looks odd, no data transfer occurs.

**Comment by kaixinjxq** [comment]
@Honry, many thanks for your review! I just submit a commit to address your comments, PTAL.

**Review by Honry — APPROVED** [review]
LGTM

**Pull request merged**